apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -195,10 +187,12 @@ const sendSubmitMeterReadingsPushNotifications = async () => {
         })
 
         // Filter meters with expired next verification date
-        const expiredVerificationMetersWithResident = metersWithResident.filter(({ meter }) => {
-            const nextVerificationDate = dayjs(meter.nextVerificationDate)
-            return dayjs().unix() >= nextVerificationDate.unix()
-        })
+        const expiredVerificationMetersWithResident = metersWithResident
+            .filter(({ meter }) => !isNull(meter.nextVerificationDate))
```

Comment:
```
            .filter(({ meter }) => {
                if (isNull(meter.nextVerificationDate)) return false

                return dayjs().unix() >= dayjs(meter.nextVerificationDate).unix()
            })
```

Reply:
What the difference? I just make filtering of empty nextVerificationDate in the separate statement

Reply:
1 cycle instead of two